### PR TITLE
Fixed bobs change

### PR DESCRIPTION
**Pull request by cmyers182002**

### CODE REVIEW

File: bobs_change.py [hunk 1]
```diff
@@ -33,5 +33,11 @@ def run_tests():
 
 # Edit this code:
 def bobs_change(money):
+    if money > 50:
+        print("sorry bob, you dont have enough")
+    if money money == 50:
```
Comment (andrewcharlton) [inline bobs_change.py]: Why is `money` here twice?

File: bobs_change.py [hunk 2]
```diff
@@ -33,5 +33,11 @@ def run_tests():
...
+        print("you've got enough")
+    if money > 50:
+        print("here is your change")
```
Comment (andrewcharlton) [inline bobs_change.py]: Make sure you're printing *how much* change bob gets here.

**Comment by andrewcharlton** [comment]
You still need to fix the six_or_bust file as well.

File: bobs_change.py (hunk 1 above)
Comment (cmyers182002) [inline bobs_change.py]: I have took away the extra 'money'

**Pull request closed without merging**